app/src/files/FilesController.js

```diff
@@ -66,9 +68,18 @@ function FilesController($scope, $injector, filesService, $translate, $mdDialog,
             templateUrl: 'app/src/files/view/addFiles.html',
             parent: angular.element(document.body),
             targetEvent: ev,
-            clickOutsideToClose: true
+            clickOutsideToClose: true,
+            locals: {
+                params: {
```

Comment:
I guess would be better call it filesUploadService or similar instead of params.